realm/realm-library/src/main/java/io/realm/ImmutableRealmSchema.java

```diff
@@ -43,7 +43,7 @@ public RealmObjectSchema get(String className) {
 
         String internalClassName = Table.getTableNameForClass(className);
         if (!realm.getSharedRealm().hasTable(internalClassName)) { return null; }
-        Table table = realm.getSharedRealm().getTable(internalClassName);
+        Table table = realm.getSharedRealm().getTable(internalClassName, realm.getSchema().getKeyPathMapping());
```

Comment:
Haven't looked up the `getTable` definition if there is still a version without a mapping. If so, maybe consider temporarily removing the version without a mapping to ensure that you don't forget to pass the mapping in somewhere.

Reply:
It is being used in some tests. I have removed the getTable without mapping to remember that we shall be passing one.